### PR TITLE
[NNS] Fix tflite subplugin so name and add unittest for tflite subplugin

### PR DESCRIPTION
**Pull request by anyj0527**
- Change `libnnstreamer_filter_tensorflow-lite` to `libnnstreamer_filter_tensorflow1-lite`
- Add `unittest_tizen_tensorflow_lite` and `unittest_tizen_tensorflow_lite-set`

**Self evaluation:**
1. Build test: [X]Passed [ ]Failed [ ]Skipped
2. Run test: [X]Passed [ ]Failed [ ]Skipped

### CODE REVIEW

File: lib/oeqa/runtime/cases/nnstreamer.py [hunk 1]
```diff
@@ -32,6 +32,20 @@ def test_nnstreamer(self):
         msg = " NNSTREAMER UNITTEST FAILED ( unittest_src_iio ): %s" % output
         self.assertEqual(status, 0, msg=msg)
 
+        cmd = 'export LD_LIBRARY_PATH=/usr/lib/gstreamer-1.0:$LD_LIBRARY_PATH; ' \
+        'export NNSTREAMER_SOURCE_ROOT_PATH=/usr/lib/nnstreamer/unittest;' \
+        '/usr/lib/nnstreamer/unittest/unittest_tizen_tensorflow_lite --gst-plugin-path=/usr/lib/gstreamer-1.0'
```
Comment (wooksong) [inline lib/oeqa/runtime/cases/nnstreamer.py:37]:
```suggestion
        cmd = 'LD_LIBRARY_PATH=/usr/lib/gstreamer-1.0:$LD_LIBRARY_PATH ' \
        'NNSTREAMER_SOURCE_ROOT_PATH=/usr/lib/nnstreamer/unittest ' \
        '/usr/lib/nnstreamer/unittest/unittest_tizen_tensorflow_lite --gst-plugin-path=/usr/lib/gstreamer-1.0'
```

How about changing it not to use ```export```?

**Review by myungjoo — APPROVED** [review]
LGTM

File: lib/oeqa/runtime/cases/nnstreamer.py (hunk 1 above)
Comment (anyj0527) [reply lib/oeqa/runtime/cases/nnstreamer.py]: Not to use `export` seems good. I updated the commit.

**Review by wooksong — APPROVED** [review]
LGTM

**Pull request merged**